### DISCUSSION TITLE
Precompiled opentherm

### DISCUSSION THREAD

**Discussion by bkbartk** [arendst/Tasmota#16532] [Q&A] [answered]
Hello, 

I know opentherm is not available as an official or unofficial Precompiled file. I use this one on one of my esp32 devices. 
Is it possible to create Precompiled file? Or do I neer to create a file manually and then upload? As of now I’m a few versions behind as this manual compilation takes more work. 

Thank you.

**Comment by Jason2866** [comment, marked as answer]
If you use with a Esp32 i could add to a unofficial variant.

**Comment by bkbartk** [comment]
I do use esp32
